src/pages/workspaces/workspace/Analyses.js

```diff
@@ -477,14 +477,23 @@ const Analyses = _.flow(
           workspace,
           onDismiss: () => setExportingAnalysisName(undefined)
         }),
-        deletingAnalysisName && h(AnalysisDeleter, {
-          printName: getDisplayName(deletingAnalysisName), googleProject, bucketName,
-          toolLabel: getTool(deletingAnalysisName),
-          onDismiss: () => setDeletingAnalysisName(undefined),
-          onSuccess: () => {
-            setDeletingAnalysisName(undefined)
-            refreshAnalyses()
-          }
+        deletingAnalysisName && h(DeleteConfirmationModal, {
+          objectType: 'analysis',
```

Comment:
Hmm.. not critical but would it be possible to use the analysis extension intelligently here so it doesn't say 'delete analysis'? 

Something like the `getExtension` function so that it says `Delete .rmd` or `Delete .ipynb`